src/app/core/services/navigation/group-nav-tree.service.ts

```diff
@@ -23,13 +24,17 @@ export class GroupNavTreeService extends NavTreeService<GroupInfo> {
     super(currentContent);
   }
 
-  childrenNavData(): OperatorFunction<GroupInfo|undefined,NavTreeElement[]> {
+  childrenNavData(): OperatorFunction<GroupInfo|undefined,NavTreeElement[]|Error> {
     return pipe(
       distinctUntilChanged((g1, g2) => g1?.route.id === g2?.route.id),
+      repeatLatestWhen(this.reload$),
       switchMap(group => {
-        if (!group || group.route.isUser) return EMPTY;
-        return this.groupNavigationService.getGroupNavigation(group.route.id).pipe(map(data => this.mapNavData(data).elements));
-      })
+        if (!group || group.route.isUser) return of([]);
```

Comment:
Just reading this, without trying to understand the reason, it is a bad smell, as you cannot really consider that non-group our user has 0 children, this just don't make sense. Maybe returning `undefined` would make more sense if you really need something to be emitted.